### PR TITLE
fix(web): premium polish pass for hero, graph, and spacing

### PR DESCRIPTION
**Pull request by Oluwatobi-Mustapha**
## Summary\n- redesign hero graph panel/connectors for cleaner premium motion and layout\n- fix short risk-scan form alignment and reduce section spacing gaps\n- upgrade trust-graph demo connectors to live animated svg paths\n- align final CTA actions and tighten global spacing mismatches\n\n## Validation\n- npm run build\n- npm run test -- --run

---
## Summary by cubic
Polished the hero and trust graph with live SVG edges, clearer node layout, and smoother motion. Tightened section spacing and aligned the short risk-scan form and final CTA.

- **UI Improvements**
  - Trust graph demo: added positioned nodes with x/y, SVG edge paths with gradient glow and animated tracers, and connected-edge highlighting; removed hard-coded span connectors.
  - Hero preview: improved edge curves with glow and tracers, streamlined to a single active layer, and added depth planes for subtle dimensionality.
  - Layout: introduced `idt-section-tight` and `idt-section-demo` for tighter spacing; styled the final CTA with `.idt-final-cta-link`.
  - Accessibility: honors reduced motion by disabling edge animations and hero tracers when `prefers-reduced-motion` is set.

- **Bug Fixes**
  - Fixed short risk-scan form alignment and spacing (updated grid, input heights, and select styling); aligned “How it works” section spacing.

<sup>Written for commit 091de1cb361f167783687063b7e531fd21af44ba. Summary will update on new commits.</sup>

### CODE REVIEW

File: web/src/styles.css [hunk 1]
```diff
@@ -563,29 +548,42 @@ h3 {
...
+  stroke-dasharray: 6 8;
+}
+
+.idt-hero-arrows g.is-high .idt-hero-arrow-glow {
+  opacity: 0.22;
 }
 
-.idt-hero-layer.is-dimmed .idt-hero-arrow {
-  animation-play-state: paused;
+.idt-hero-arrow-tracer {
```
[the remark this answers is not in the document]
Comment (Oluwatobi-Mustapha) [reply web/src/styles.css:575]: Added reduced-motion fallback for the hero tracer in `web/src/styles.css` by hiding `.idt-hero-arrow-tracer` under `@media (prefers-reduced-motion: reduce)`.

This complements the React-level `!reducedMotion` guard in `HeroProductReveal` and prevents motion during initial paint or non-React edge cases.

Included in commit `091de1c` and verified with `npm run test -- --run` (23 tests passed).

**Review by John-Reese01 — APPROVED** [review]
